packages/angular_devkit/build_angular/src/webpack/plugins/any-component-style-budget-checker.ts

```diff
@@ -20,16 +21,17 @@ const PLUGIN_NAME = 'AnyComponentStyleBudgetChecker';
  */
 export class AnyComponentStyleBudgetChecker {
   private readonly budgets: Budget[];
+
   constructor(budgets: Budget[]) {
     this.budgets = budgets.filter((budget) => budget.type === Type.AnyComponentStyle);
   }
 
   apply(compiler: Compiler) {
-    compiler.hooks.compilation.tap(PLUGIN_NAME, (compilation) => {
-      compilation.hooks.afterOptimizeChunkAssets.tap(PLUGIN_NAME, () => {
+    compiler.hooks.compilation.tap(PLUGIN_NAME, (_compilation) => {
```

Comment:
Can you rename this to something else? Typically parameters suffixed with `_` signifies that the parameter is unused.